api/games/augment.js

```diff
@@ -27,10 +29,10 @@ export const augmentGameObject = ({
     players: {
       get() {
         if (!gamePlayers) {
-          gamePlayers = game.getPlayers();
+          gamePlayers = Players.find({ _id: { $in: game.playerIds } }).fetch();
 
           if (firstRoundId) {
-            round = game.getRounds().find(r => r._id === firstRoundId);
+            round = Rounds.find({ firstRoundId }).fetch();
```

Comment:
Is this correct? Shouldn't it be something like:
```suggestion
            round = Rounds.findOne(firstRoundId).fetch();
```

Reply:
My apologize , i have fixed that one.